anthos-onprem-terraform/avmw_user_cluster_metallb/README.md

```diff
@@ -58,20 +58,31 @@ have changed directory to where this samples is:
 
 ### Upgrade the user cluster with terraform
 
-Before upgrading the user cluster, please make sure the admin cluster platform
-controller has been upgraded to the target version. The steps to upgrade the
-admin cluster platform controller is listed in the
-[public documentation](https://cloud.google.com/anthos/clusters/docs/on-prem/latest/how-to/upgrade-on-prem-api#available_versions_for_upgrades).
+Before upgrading the user cluster, please make sure the admin cluster has been 
+enrolled in the Anthos On-Prem API. Steps for enrolling the admin cluster are
```

Comment:
do we need to mention anything about the cloud controller here, since we are looking to launch that soon after which this step will become no-op.